test/TorchSharpTest/NN.cs

```diff
@@ -325,6 +325,8 @@ public void TestIdentity()
                 var input = torch.randn(new long[] { 1, 1000 }, device: device);
                 var output = lin.call(input);
 
+                output[0, 511] = 10; // When we modify the copy, the original should be altered, too.
+
                 Assert.Equal(device.type, output.device_type);
```

Comment:
Is this how `Identity` behaves in PyTorch? o-O

Reply:
Yup:

```Python
>>> id = torch.nn.Identity()
>>> input = torch.zeros(10,10)
>>> output = id(input)
>>> output[0,0] = 13
>>> input
tensor([[13.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.],
        [ 0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.,  0.]])
```

Reply:
From the PyTorch source code:

```Python
    def forward(self, input: Tensor) -> Tensor:
        return input
```